### PR TITLE
Loki: Add `lokiQuerySplittingConfig` feature flag

### PR DESCRIPTION
**Pull request by svennergr**
**What is this feature?**

This PR adds the `lokiQueryChunkingOption` feature flag to show a per query option in the Loki datasource to configure chunk durations.

### CODE REVIEW

**Review by matyax — CHANGES_REQUESTED** [review]
Looks good! Added one comment about the name of the flag.

File: docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Alpha features might be changed or removed without prior notice.
 | `editPanelCSVDragAndDrop`          | Enables drag and drop for CSV and Excel files                                                                                                                                |
 | `logsContextDatasourceUi`          | Allow datasource to provide custom UI for context view                                                                                                                       |
 | `lokiQuerySplitting`               | Split large interval queries into subqueries with smaller time intervals                                                                                                     |
+| `lokiQueryChunkingOption`          | Give users the option to configure chunk durations for Loki queries                                                                                                          |
```
[the remark this answers is not in the document]
Comment (svennergr) [reply docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md:91]: Changed in b0287e5081b6b26e111468c8de49366a348d5b72

[the remark this answers is not in the document]
Comment (svennergr) [reply docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md]: a197c5f2a10ae1109799af1bacab445054a4ed19 🙈

**Review by matyax — APPROVED** [review]
👌

**Pull request merged**